readme.txt

```diff
@@ -20,8 +21,47 @@ For more information, see:
 * https://www.classicpress.net
 * https://github.com/ClassicPress/ClassicPress-Migration-Plugin
 
+== Installation ==
+
+To install a fresh version of ClassicPress:
+
+To install ClassicPress on a current WordPress site (switch a current WordPress site to ClassicPress):
+
+1. Backup the current site. You can do a manual backup in your hosting panel and export the database or you can use a backup plugin to do so. **PLEASE NOTE:** As ClassicPress is currently in Alpha, we do not recommend that you switch a live production site to ClassicPress.
+2. Download the ClassicPress Migration plugin from this link:  Insert Link
+3. Upload the plugin to the current WordPress plugins section of your site.
+4. Activate the ClassicPress Migration Plugin.
+5. Go to "Settings" of the ClassicPress Migration plugin at Tools->Switch to ClassicPress. All pre-flight checks must be marked in green to proceed with the switch.
+6. Press the "Switch To ClassicPress" button.
+7. This will take between one or two minutes so grab some water 🙂
```

Comment:
"This will take between one or two minutes" - we don't know that, on the sites where I've tested this works almost immediately.  How about this instead: "This may take a few minutes depending on your hosting provider, so go grab some water or a beverage of your choice :)"

Reply:
> "This will take between one or two minutes" - we don't know that, on the sites where I've tested this works almost immediately. How about this instead: "This may take a few minutes depending on your hosting provider, so go grab some water or a beverage of your choice :)"

Agreed -- basically just pulled this from the site, so happy to change it